mk_verificator/benchmarks/engine/runner.py

```diff
@@ -0,0 +1,41 @@
+import multiprocessing as mp
```

Comment:
let's keep it because of GIL, that's not so critical for us for now and for perspective too